module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIni.php

```diff
@@ -117,6 +117,11 @@ public function addToPathStack($pathStack)
      */
     public function load($locale, $filename)
     {
+
```

Comment:
Unnecessary whitespace at top of function (I'm not sure why php-cs-fixer isn't complaining about this).

Reply:
Fixed